gradle.properties

```diff
@@ -0,0 +1,5 @@
+description = "Micro-service exposing API CRUD operations for assessment items stored in the item bank."
+
+appGroup = org.opentestsystem.ap
+appArtifact = ims
+appVersion = 0.0.1-SNAPSHOT
```

Comment:
Gradle reads this file and adds all the properties to the project which can be referenced from build.gradle.